src/routes/[...lang=locale]/anime/+page.svelte

```diff
@@ -334,52 +342,62 @@
-									<span class="line-clamp-1 font-bold text-accent md:text-[1.15vw]"
-										>{episode.title}</span
-									>
-									<div
-										class="flex items-center font-semibold md:gap-[0.5vw] md:text-[0.8vw] md:leading-none"
-									>
-										<span class="whitespace-nowrap"
-											>Ep {episode.ep_number.toString().padStart(2, "0")}</span
+						{#if image_loaded}
+							<div
+								in:blur
+								class="relative w-full snap-start border border-[var(--border-color)] bg-cover bg-center duration-300 [background-image:var(--background-image)] md:h-[5vw] md:rounded-[0.75vw] md:border-[0.15vw]"
+								style="--background-image: url({episode.banner}); --border-color: #{color_palette};"
+							>
+								<div class="absolute inset-0 bg-secondary/75 md:rounded-[0.75vw]"></div>
+								<div class="relative flex size-full items-center md:gap-[1vw] md:p-[0.5vw]">
+									<img
+										src={episode.cover}
+										alt=""
+										class="h-full object-cover object-center md:w-[2.5vw] md:rounded-[0.5vw]"
+									/>
+									<div class="flex flex-1 flex-col md:gap-[0.15vw]">
+										<span class="line-clamp-1 font-bold text-accent md:text-[1.15vw]"
+											>{episode.title}</span
 										>
-										<Circle class="opacity-75 md:size-[0.25vw]" />
-										<span class="line-clamp-1">{episode.timestamp}</span>
+										<div
+											class="flex items-center font-semibold md:gap-[0.5vw] md:text-[0.8vw] md:leading-none"
+										>
+											<span class="whitespace-nowrap"
+												>Ep {episode.ep_number.toString().padStart(2, "0")}</span
+											>
+											<Circle class="opacity-75 md:size-[0.25vw]" />
+											<span class="line-clamp-1">{episode.timestamp}</span>
+										</div>
 									</div>
+									<a
+										href="anime/mal/{episode.id}/episode/{episode.ep_number}"
+										class="btn h-max min-h-max rounded-full border-none bg-warning [background-color:var(--border-color)] md:mr-[0.5vw] md:p-[0.75vw]"
+									>
+										<Play class="md:size-[1.25vw]" />
+									</a>
 								</div>
-								<a
-									href="anime/mal/{episode.id}/episode/{episode.ep_number}"
-									class="btn h-max min-h-max rounded-full border-none bg-warning md:mr-[0.5vw] md:p-[0.75vw]"
-									style="background-color: #{color_palette};"
-								>
-									<Play class="md:size-[1.25vw]" />
-								</a>
 							</div>
-						</div>
+						{:else}
+							<div
+								class="flex items-center bg-neutral/25 md:h-[5vw] md:gap-[1vw] md:rounded-[0.75vw] md:p-[0.5vw]"
+							>
+								<div class="h-full bg-neutral md:w-[2.5vw] md:rounded-[0.5vw]"></div>
+								<div class="flex flex-1 flex-col md:gap-[0.5vw]">
+									<div class="rounded-full bg-neutral md:h-[1vw]"></div>
+									<div class="w-1/2 rounded-full bg-neutral md:h-[0.5vw]"></div>
+								</div>
+								<div class="rounded-full bg-neutral md:mr-[0.5vw] md:size-[2.75vw]"></div>
+							</div>
+						{/if}
```

Comment:
We want transition here

Reply:
It has transition, take a close look

Reply:
use `blur`

Reply:
```svelte
{:else}
							<div
								class="flex items-center bg-neutral/25 md:h-[5vw] md:gap-[1vw] md:rounded-[0.75vw] md:p-[0.5vw]"
							>
								<div class="h-full bg-neutral md:w-[2.5vw] md:rounded-[0.5vw]"></div>
								<div class="flex flex-1 flex-col md:gap-[0.5vw]">
									<div class="rounded-full bg-neutral md:h-[1vw]"></div>
									<div class="w-1/2 rounded-full bg-neutral md:h-[0.5vw]"></div>
								</div>
								<div class="rounded-full bg-neutral md:mr-[0.5vw] md:size-[2.75vw]"></div>
							</div>
						{/if}
						```
On this

Reply:
> Use a grid on this

Even tho, output is same...
and this is unnecessary in this case